.github/workflows/release.yml

```diff
@@ -18,22 +18,6 @@ jobs:
         env:
           CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
 
-  x86-64-apple-darwin-release:
-    name: Build and upload x86-64-apple-darwin-release to Cloudsmith
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v1
-      - name: brew install ponyc
-        run: brew install ponyc
-      - name: brew install dependencies
-        run: brew install coreutils python
-      - name: pip install dependencies
-        run: pip3 install --upgrade cloudsmith-cli
-      - name: Build and upload
-        run: bash .ci-scripts/release/x86-64-apple-darwin-release.bash
-        env:
-          CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
-
```

Comment:
The `x86-64-apple-darwin-release` needs to be removed below in `needs: [x86-64-unknown-linux-release, x86-64-apple-darwin-release, build-release-docker-images]`

Reply:
updated